test/extended/operators/olm.go

```diff
@@ -105,4 +106,26 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 		}
 	})
 
+	// OCP-23395 - Deleted catalog registry pods and verify if them are recreated automatically
+	// author: bandrade@redhat.com
+	g.It("Recreate catalog registry pods and if them are deleted", func() {
+		msg, err := oc.SetNamespace("openshift-operator-lifecycle-manager").AsAdmin().Run("delete").Args("pods", "-l app=catalog-operator").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(msg).To(o.ContainSubstring("deleted"))
+
+		error := wait.Poll(5*time.Second, 60*time.Second, func() (bool, error) {
```

Comment:
`error` is a reserved word -- try something like `pollErr` instead.

Reply:
changed as requested